_posts/2019-10-01-intro-to-hmc/hmc-intro.Rmd

```diff
@@ -0,0 +1,284 @@
+---
+title: "On leapfrogs, crashing satellites, and going nuts: A very first conceptual introduction to Hamiltonian Monte Carlo"
+description: > 
+  TensorFlow Probability, and its R wrapper tfprobability, provide Markov Chain Monte Carlo (MCMC) methods that were used in a number of recent posts on this blog. These posts were directed to users already comfortable with the method, and terminology, per se, which readers mainly interested in deep learning won't necessarily be. Here we try to make up leeway, introducing Hamitonian Monte Carlo (HMC) as well as a few often-heard "buzzwords" accompanying it, always striving to keep in mind what it is all "for".
+author:
+  - name: Sigrid Keydana
+    affiliation: RStudio
+    affiliation_url: https://www.rstudio.com/
+slug: keydana2019hmc
+date: 10-01-2019
+categories:
+  - Probability and statistics
+  - Concepts
+  - Introductions
+bibliography: bibliography.bib
+output:
+  distill::distill_article:
+    self_contained: false
+preview: images/mb.png
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, eval = FALSE)
+```
+
+Why a _very (meaning: VERY!) first conceptual introduction to Hamiltonian Monte Carlo_ (HMC) on this blog? 
+
+Well, in our endeavor to feature the various capabilities of TensorFlow Probability (TFP) / [tfprobability](https://rstudio.github.io/tfprobability/), we started showing examples of how to fit hierarchical models, using one of TFP's joint distribution classes ^[[tfd_joint_distribution_sequential](https://rstudio.github.io/tfprobability/reference/tfd_joint_distribution_sequential.html)] and HMC. The technical aspects being complex enough in themselves, we never gave an introduction to the "math side of things". Here we are trying to make up for this. 
```

Comment:
Should we add references to https://blogs.rstudio.com/tensorflow/posts/2019-06-25-dynamic_linear_models_tfprobability/ and https://blogs.rstudio.com/tensorflow/posts/2019-05-06-tadpoles-on-tensorflow/ after HMC? It would help for readers that want to jump to the application of HMC before understanding how it works.

Reply:
good idea, i'll add them in a footnote!